Tasks/Common/packaging-common/locationUtilities.ts

```diff
@@ -94,7 +94,7 @@ export async function getPackagingUris(protocolType: ProtocolType): Promise<Pack
 
     tl.debug('Acquiring Packaging endpoints from ' + serviceUri);
 
-    const connectionData = await locationApi.getConnectionData(interfaces.ConnectOptions.IncludeServices);
+    const connectionData = await retryOnExceptionHelper(() => locationApi.getConnectionData(interfaces.ConnectOptions.IncludeServices), 5, 1000);
```

Comment:
i thought that the node api already does retries under the covers (for known status codes). For these known case does this mean we will now retry 5x5 times (assuming 5 is what is under covers?)

Reply:
Yes, that's correct.